src/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -256,6 +256,9 @@
     <Compile Include="$(CommonPath)\System\Security\Cryptography\Asn1\EncryptedPrivateKeyInfoAsn.cs">
       <Link>Common\System\Security\Cryptography\Asn1\EncryptedPrivateKeyInfoAsn.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Security\Cryptography\Asn1\GeneralNameAsn.cs">
+      <Link>Common\System\Security\Cryptography\Asn1\GeneralNameAsn.cs</Link>
```

Comment:
I have mixed feelings about the product code and the test code sharing source.  In the "everything is great" case it's a benefit, but it loses the part where the tests let you know that you made a change in the product (in case you didn't mean to).

What's the motivation here?  Is there a bug?  Is it just general tidiness?